src/selectorNameBase.ts

```diff
@@ -69,7 +69,7 @@ class SelectorNameValidatorWalker extends Lint.RuleWalker {
   }
 
   private validateSelector(className: string, arg: ts.Node) {
-    if (arg.kind === SyntaxKind.current().ObjectLiteralExpression) {
+    if (arg && arg.kind === SyntaxKind.current().ObjectLiteralExpression) {
```

Comment:
Say I could provide some code, I have 200+ files, how would I know which one is causing this? I cannot just give you access to everything. Maybe you can point me to some things I could look at in the components that could cause this? It might be that is not even related to the actual components but the linter or TS.